apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx

```diff
@@ -81,9 +85,38 @@ The above command will migrate your data to the latest schema. This is a crucial
 
 5. That's it! Once the migration is complete, you can **now access your Formbricks instance** at the same URL as before.
 
+#### Restoring the database after a failed upgrade
+
+Replace the path to the `.dump` file and run:
+
+<Col>
+<CodeGroup title="Restore the database">
+
+```bash
+docker exec -i formbricks-quickstart-postgres-1 pg_restore --clean -U postgres -v -d formbricks < formbricks_pre_v1.6_20240320_123522.dump
```

Comment:
Since the timestamp in `formbricks_pre_v1.6_20240320_123522` can change for each user, can we rewrite this to mention that a file similar to ....